client/src/RatingsAndReviews/RatingsBreakdown/RatingBreakdown.jsx

```diff
@@ -10,13 +10,24 @@ const RatingBreakdown = (props) => {
       <BreakdownClickable stars={3} percent={props.percentages[2]} count={props.counts[2]} starFilterClick={props.starFilterClick}/>
       <BreakdownClickable stars={2} percent={props.percentages[1]} count={props.counts[1]} starFilterClick={props.starFilterClick}/>
       <BreakdownClickable stars={1} percent={props.percentages[0]} count={props.counts[0]} starFilterClick={props.starFilterClick}/>
+      <div
+        className="sort-options-selected"
+        style={{display: props.starFilters.length === 0 ? "none" : "block"}}>
+        <div
+          className="sort-option-title"
+          remove="all"
```

Comment:
Had not seen a remove property before, don't totally understand it but I bet you know what you're doing here